master-firmware/src/strategy.cpp

```diff
@@ -42,6 +42,12 @@ static void wait_for_autoposition_signal(void)
     wait_for_starter();
 }
 
+void strategy_stop_robot(struct _robot* robot)
+{
+    trajectory_hardstop(&robot->traj);
```

Comment:
Is this enough to stop the robot? I think we discussed that it was not working once.